CheckSammy.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```

Comment:
This is an assumption - are you, in fact, using Python 3.6?  
(If so, you can use [string interpolation](https://www.python.org/dev/peps/pep-0498/), which was introduced in that version. Not my favourite string interpolation syntax, but nicer than `%`-strings, IMO).